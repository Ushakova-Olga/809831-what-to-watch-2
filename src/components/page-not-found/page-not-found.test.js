import React from "react";
import renderer from "react-test-renderer";
import PageNotFound from "./page-not-found";

it(`Page not found component correctly renders`, () => {
  const tree = renderer.create(<PageNotFound/>).toJSON();
  expect(tree).toMatchSnapshot();
});
