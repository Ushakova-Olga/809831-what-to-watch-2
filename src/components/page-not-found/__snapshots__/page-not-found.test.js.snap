// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page not found component correctly renders 1`] = `
<h1
  style={
    Object {
      "marginTop": "200px",
      "textAlign": "center",
    }
  }
>
  404: Not Found
</h1>
`;
